src/moveFiles.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function normalizePaths(source, destination) {
+  const currentPath = path.join(__dirname, source);
+  let newPath = '';
+
+  if (destination.endsWith('/')) {
+    const dirPath = path.join(__dirname, destination);
+
+    newPath = path.join(dirPath, source);
+  } else {
+    if (destination.includes('./')) {
+      newPath = path.join(destination, source);
+    } else if (!destination.includes('/') && !destination.includes('.')) {
+      newPath = path.join(__dirname, destination, source);
+
+      // if directory is not exist, we rename the file
```

Comment:
Remove comment. Try to write obvious code with meaningful variable and function naming